rewrite-java-lombok/src/main/java/org/openrewrite/java/lombok/Lombok.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.lombok;
+
+import org.jspecify.annotations.Nullable;
+import org.openrewrite.internal.ReflectionUtils;
+
+import javax.annotation.processing.Processor;
+import java.io.File;
+import java.lang.reflect.Constructor;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+
+public class Lombok {
```

Comment:
I would rename this class to `LombokProcessorInitializer` + rename `createLombokProcessor` to `initialize`.